src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
Remove extra wrapper?
Put the answer in this comment, why do we need React fragment

Reply:
Fragment needed when we have to use 2 or more parent tags
my bad, i didn't see that there only 1 parent div